numpy/core/multiarray.py

```diff
@@ -1107,6 +1107,21 @@ def copyto(dst, src, casting=None, where=None):
         A boolean array which is broadcasted to match the dimensions
         of `dst`, and selects elements to copy from `src` to `dst`
         wherever it contains the value True.
+
+    Examples
+    --------
+    >>> A = np.array([4,5,6])
+    >>> B = [1,2,3]
```

Comment:
Please follow PEP-8 python formatting, in practice for this pull request means to add a space after each coma.
```suggestion
    >>> B = [1, 2, 3]
```